ipamutils/utils.go

```diff
@@ -37,18 +37,37 @@ type NetworkToSplit struct {
 // InitNetworks initializes the broad network pool and the granular network pool
```

Comment:
Please change 'broad network' and 'granular network' to 'local scope' and 'global scope' to match the new variable names.